The release tarball does not include requirements for tests and tries
to fetch them. Override so can handle with SRC_URI+DEPEND instead.
--- a/ext/c4core/cmake/c4Project.cmake
+++ b/ext/c4core/cmake/c4Project.cmake
@@ -2902,4 +2902,7 @@
         c4_log("testing requires googletest")
         if(NOT TARGET gtest)
+            find_package(GTest REQUIRED)
+	endif()
+	if(FALSE)
             # support for old gcc-4.8 and 4.9
             if((CMAKE_CXX_COMPILER_ID STREQUAL "GNU") AND
--- a/ext/testbm.cmake
+++ b/ext/testbm.cmake
@@ -1,4 +1,2 @@
 # these are used both for testing and benchmarking
-c4_require_subproject(c4fs REMOTE
-    GIT_REPOSITORY https://github.com/biojppm/c4fs
-    GIT_TAG master)
+c4_require_subproject(c4fs SUBDIRECTORY ext/c4fs)
--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -158,12 +158,8 @@
     set(ed ${CMAKE_CURRENT_BINARY_DIR}/subprojects) # casual ryml extern dir (these projects are not part of ryml and are downloaded and compiled on the fly)
 
-    c4_require_subproject(c4log REMOTE
-        GIT_REPOSITORY https://github.com/biojppm/c4log
-        GIT_TAG master)
+    c4_require_subproject(c4log SUBDIRECTORY ../ext/c4log)
 
     set(tsdir ${ed}/yaml-test-suite)
-    c4_download_remote_proj(yaml-test-suite suite_dir
-        GIT_REPOSITORY https://github.com/yaml/yaml-test-suite
-        GIT_TAG data-2022-01-17)
+    set(suite_dir ${CMAKE_SOURCE_DIR}/ext/yaml-test-suite)
     if(NOT EXISTS ${suite_dir}/229Q)
         c4_err("cannot find yaml-test-suite at ${suite_dir} -- was there an error downloading the project?")
